flexmeasures/ui/crud/assets.py

```diff
@@ -148,6 +149,15 @@ def expunge_asset():
     return asset_data
 
 
+def user_can_create_assets() -> bool:
+    try:
+        check_access(current_user.account, "create-children")
+    except Exception as exc:
+        current_app.logger.error(exc)
```

Comment:
Won't this lead to an error log statement every time a non-admin user views the assets page?

Reply:
Also a good catch!